recipes/unity/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conans import ConanFile, CMake
+from conans import tools
+
+
+class UnityConan(ConanFile):
+    name = "unity"
```

Comment:
I think we should change name to something more specific, as `unity` is well-known 3d-engine, user-interface from Canonical and programming language.